pkg/injector/injector.go

```diff
@@ -120,34 +121,37 @@ func NewInjector(authUIDs []string, config Config, daprClient scheme.Interface,
 }
 
 // AllowedControllersServiceAccountUID returns an array of UID, list of allowed service account on the webhook handler.
-func AllowedControllersServiceAccountUID(ctx context.Context, kubeClient kubernetes.Interface) ([]string, error) {
-	allowedUids := []string{}
-	for i, allowedControllersServiceAccount := range allowedControllersServiceAccounts {
-		saUUID, err := getServiceAccount(ctx, kubeClient, allowedControllersServiceAccount)
-		// i == 0 => "replicaset-controller" is the only one mandatory
-		if err != nil {
-			if i == 0 {
-				return nil, err
-			}
-			log.Warnf("Unable to get SA %s UID (%s)", allowedControllersServiceAccount, err)
-			continue
-		}
-		allowedUids = append(allowedUids, saUUID)
-	}
+func AllowedControllersServiceAccountUID(ctx context.Context, cfg Config, kubeClient kubernetes.Interface) ([]string, error) {
+	allowedList := strings.Split(cfg.AllowedServiceAccountInfos, ",")
+	allowedList = append(allowedList, AllowedServiceAccountInfos...)
 
-	return allowedUids, nil
+	return getServiceAccount(ctx, kubeClient, allowedList)
 }
 
-func getServiceAccount(ctx context.Context, kubeClient kubernetes.Interface, allowedControllersServiceAccount string) (string, error) {
+// getServiceAccount parses "service-account:namespace" k/v list and returns an array of UID.
+func getServiceAccount(ctx context.Context, kubeClient kubernetes.Interface, allowedServiceAcccountInfos []string) ([]string, error) {
 	ctxWithTimeout, cancel := context.WithTimeout(ctx, getKubernetesServiceAccountTimeoutSeconds*time.Second)
 	defer cancel()
 
-	sa, err := kubeClient.CoreV1().ServiceAccounts(metav1.NamespaceSystem).Get(ctxWithTimeout, allowedControllersServiceAccount, metav1.GetOptions{})
+	serviceaccounts, err := kubeClient.CoreV1().ServiceAccounts("").List(ctxWithTimeout, metav1.ListOptions{})
```

Comment:
@daixiang0 you probably need to add the `list` verb permission here: https://github.com/dapr/dapr/blob/5753fa83b7bcfde35b322a386d7dfdb2ea46371c/charts/dapr/charts/dapr_rbac/templates/ClusterRoleBinding.yaml#L37.

Reply:
Thanks, I forget it.